app/services/estate_visit_query.rb

```diff
@@ -35,7 +31,23 @@ def processed(limit:, prisoner_number: nil)
   def inbox_count
     Visit.
       ready_for_processing.
-      from_estate(@estate).
+      from_estates(@estates).
       count
   end
+
+private
+
+  def grouped_visits(visits)
```

Comment:
:scream: could we make this readable, i'm thinking you can get this same result using a postgres window function? I know that the intent here is to reuse the the  visits in memory array but this is really hard to read.